### PR TITLE
HBASE-17962 Improve documentation on Rest interface

### PR DESCRIPTION
**Pull request by nielsbasjes**
The simple improvement of the mentioned documentation.

### CODE REVIEW

File: src/main/asciidoc/_chapters/external_apis.adoc [hunk 1]
```diff
@@ -225,14 +225,17 @@ creation or mutation, and `DELETE` for deletion.
 |Description
 |Example
 
-|/_table_/_row_/_column:qualifier_/_timestamp_
+|/_table_/_row_
 |GET
-|Get the value of a single row. Values are Base-64 encoded.
+|Get the all columns of a single row. Values are Base-64 encoded. This requires the "Accept" request header with a type that can hold multiple columns (like xml, json or protobuf).
```
Comment (busbey) [inline src/main/asciidoc/_chapters/external_apis.adoc]: Don't need the "the" prior to "all columns"

File: src/main/asciidoc/_chapters/external_apis.adoc [hunk 2]
```diff
@@ -225,14 +225,17 @@ creation or mutation, and `DELETE` for deletion.
...
 |curl -vi -X GET \
   -H "Accept: text/xml" \
   "http://example.com:8000/users/row1"
 
-curl -vi -X GET \
+|/_table_/_row_/_column:qualifier_/_timestamp_
+|GET
+|Get the single value of a single column. Values are Base-64 encoded.
```
Comment (busbey) [inline src/main/asciidoc/_chapters/external_apis.adoc]: "Get the value of a single column" would be clearer.

**Comment by busbey** [comment]
this is a great improvement, thanks!

would you mind also squashing your commits into one when you update?

**Comment by nielsbasjes** [comment]
@busbey I implemented the changes, did a squash and a force push into here.
Please verify.

**Comment by nielsbasjes** [comment]
@busbey: Looks to me that this pull request can be closed now. Correct?

**Comment by busbey** [comment]
Yep!

Sorry, I forgot to include the github magic comment in the commit message.

**Pull request closed without merging**